Makefile

```diff
@@ -141,3 +147,29 @@ $(call add-crd-gen,tuned,$(API_TYPES_DIR),./manifests,./manifests)
 $(call add-profile-manifests,manifests,./profile-patches,./manifests)
 
 .PHONY: all build deepcopy crd-schema-gen test-e2e verify verify-gofmt clean local-image local-image-push
+
+# Performance addons operations
```

Comment:
Nit: Above you use PAO as a comment, why use "Performance addons" here and a new line?